sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -1324,29 +1324,41 @@ export class CommandStem<A extends Args[] | { [argName: string]: any } = [] | {}
       models: this._models,
       metadata: this._metadata,
       description: description,
-      ...(this.absoluteTime && { absoluteTime: this.absoluteTime }),
-      ...(this.epochTime && { epochTime: this.epochTime }),
-      ...(this.relativeTime && { relativeTime: this.relativeTime }),
+      ...(this._absoluteTime && { absoluteTime: this._absoluteTime }),
+      ...(this._epochTime && { epochTime: this._epochTime }),
+      ...(this._relativeTime && { relativeTime: this._relativeTime }),
     });
   }
   // @ts-ignore : 'Description' found in JSON Spec
   public GET_DESCRIPTION(): Description | undefined {
     return this._description;
   }
 
+  public GET_ABSOLUTE_TIME(): Temporal.Instant | undefined {
```

Comment:
Why the switch here?

Could you use an actual getter here? ie
```ts
public get absoluteTime(): Temporal.Instant | undefined {
  return this._absoluteTime;
}
```

it allows you to maintain the same code elsewhere, ie step.absoluteTime

Reply:
I wanted to maintain consistency with the eDSL's design. My approach involves exposing a limited set of methods during chaining so the user can just focus on the CAPITALIZE methods and ignore the other values that show up in the context assist. ex:

BAKE_BANANA.<a subset of methods>

Content assisted box:
...
MODEL(model)
GET_MODEL()
METADATA(metadata)
GET_METADAT()
DESCRIPTION(string)
GET_DESCRIPTION()
GET_ABSOLUTE_TIME()
GET_EPOCH_TIME()
GET_RELATIVE_TIME()
etc.
...

Reply:
As I move forward with the custom worker, I'm considering implementing a filter to refine the content assist feature. I want to only show those methods that are CAPITALIZED, while concealing the others.

Reply:
Fair enough. I suppose if modifying the autocomplete was easier for us, we might have gone a different route, but I like the ingenuity in working with what we have!